plaidml/bridge/openvino/src/plaidml_plugin/ops/reduce_logical_or.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset1.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("reducelogicalor", [](const Context& ctx) {
+  IE_ASSERT(ctx.operands.size() == 2);
+  auto I = ctx.operands.at(0);
+  std::vector<size_t> axes = get_axis_vector_from_constant_operand(1, ctx.layer);
+  auto* layer = dynamic_cast<ngraph::opset1::ReduceLogicalAnd*>(ctx.layer);
```

Comment:
Is this right? Should it be `ReduceLogicalOr`?

Reply:
It should be `ReduceLogicalOr`

Reply:
Oops, my bad.

Reply:
thanks @flaub I'll put in a PR to fix this